core/plugins/stack/python/black.ts

```diff
@@ -0,0 +1,34 @@
+import { IntrospectFn } from "../../../types.ts";
+import { hasPythonDependency } from "./dependencies.ts";
+
+export interface Black {
+  name: "black";
+  hasConfig: boolean;
```

Comment:
Is this useful for the templates?

Reply:
Nope, I changed it here to leave the information only in the internal variable in the function